src/open_inwoner/accounts/query.py

```diff
@@ -100,7 +100,9 @@ def mark_seen(self, user, other_user) -> int:
 
 
 class InviteQuerySet(QuerySet):
-    def get_pending_invitations_for_user(self, user: "User") -> "InviteQuerySet":
+    def get_pending_invitations_for_user(
+        self, user: "User"  # noqa
```

Comment:
Another weird `noqa`?

Reply:
`src/open_inwoner/accounts/query.py:6:48: F821 undefined name 'User'`

I suppose I could also use `get_user_model` instead

Reply:
Ugh, even when importing in `if TYPE_CHECKING:`?

Reply:
Using that works indeed :+1:

Reply:
Cool!